tempesta_fw/t/functional/helpers/control.py

```diff
@@ -423,7 +497,7 @@ def servers_start(servers):
 def servers_stop(servers):
     threads = __servers_pool_size(len(servers))
     pool = multiprocessing.Pool(threads)
-    pool.map(Nginx.stop, servers)
+    pool.map(Nginx.force_stop, servers)
```

Comment:
I can't agree here. `stop()` tracks state while `force_stop()`. Why is stateless version is used here?

Reply:
fixed